clients/typescript/src/types.ts

```diff
@@ -69,6 +139,43 @@ interface PathfinderUniqueParams {
   num_threads: number;
 }
 
+/**
+ * @typedef {Object} PathfinderParams
+ * Parameters for the Pathfinder algorithm.
+ * @property {string | StanVariableInputs} [data=""] The data for the model.
+ * @property {number} [num_paths=4] The number of individual paths to run.
+ * @property {string | StanVariableInputs | string[] | StanVariableInputs[]} [inits=""]
+ * The initial values for the algorithm. If an array, must have length `num_paths`.
+ * @property {number | null} [seed] The seed for the random number generator.
```

Comment:
Doesn't this default to `null` (or `undefined`?)

Reply:
It does, but the rendering for `null` is also pretty bad